### PR TITLE
Delete duplicated boroughs

### PR DESCRIPTION
**Pull request by ibramoha**
I have deleted the copied boroughs from the list of boroughs and removed unnecessary key value pairs.

### CODE REVIEW

File: frontend/src/Data/Boroughs.json [hunk 1]
```diff
@@ -1,4154 +1,35 @@
...
-    "borough": "Ealing",
-    "postTown": "GREENFORD",
-    "postcodeDistrict": "UB6",
-    "dialCode": "020",
-    "osGridRef": "TQ165835"
-  },
-  {
-    "location": "Petersham",
-    "borough": "Richmond upon Thames",
-
```
Comment (LouiaR) [inline frontend/src/Data/Boroughs.json]: Good job:+1: @ibramoha  I think it's City of London  not City

Comment (LouiaR) [inline frontend/src/Data/Boroughs.json]: Good job:+1: @ibramoha  I think it's City of London  not City

**Comment by ibramoha** [comment]
@DwinaTech  it is one borough.

File: frontend/src/Data/Boroughs.json (hunk 1 above)
Comment (ibramoha) [reply frontend/src/Data/Boroughs.json]: Thanks!

**Pull request merged**